tests/modules/notebooks/test_notebook_stack.py

```diff
@@ -0,0 +1,18 @@
+
+def test_notebook_stack(client, sgm_notebook, group):
```

Comment:
Replaces test in`tests/api/test_stack.py`? @nikpodsh

Reply:
Yes, I extracted the part related to notebooks from `tests/api/test_stack.py` to a dedicated test